test/WebSites/RazorWebSite/Views/UrlResolution/Index.cshtml

```diff
@@ -33,6 +33,7 @@
         <menuitem icon=~/Person />
     </menu>
     <object data="~/Person" archive="~/Person/@Model.Name" data="~/Person" archive="~/Person/@Model.Name" />
+    <object archive="@if(Model.Name != null) { <text>~/Person/@Model.Name</text> } else { <text>~/Person</text> } " />
```

Comment:
:+1: to @pranavkm's suggestion though the compiler is more likely to complain about an `async` lambda _without_ an `await`. Nice that it doesn't.